minitests/litex/src.vivado/Makefile

```diff
@@ -0,0 +1,35 @@
+BASE_PATH = $(abspath $(PWD)/../../../)
+VIVADO = $(BASE_PATH)/utils/vivado.sh
+PART = xc7a35tcsg324-1
+BIT2FASM_ARGS = --db-root "$(BASE_PATH)/database/artix7" --part "$(BASE_PATH)/database/artix7/$(PART)" --verbose
```

Comment:
``-db-root`` should not be required if ``prjxray/settings/artix.sh`` was sourced.